Auth/Config/AuthenticationConfig.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Diagnostics.Contracts;
+
+namespace Core.AeroGear.Mobile.Auth.Config
+{
+    /// <summary>
+    /// Configurations for the authentication service
+    /// </summary>
+    public class AuthenticationConfig
+    {
+        /// <returns>the redirect uri used during the authentication process</returns>
+        public Uri RedirectUri { get; private set; }
+
+        /// <returns>the OIDC scopes for the auth request. If no scopes are defined, the default 'openid scope will be sent</returns>
+        public string Scopes { get; private set; }
+
+        /// <returns>the minimum time between Json Web Key Set (JWKS) requests in minutes. Default value is 1440 (1 day)</returns>
+        public UInt16 MinTimeBetweenJwksRequests { get; private set; }
```

Comment:
UInt16?  Maybe ushort?

Reply:
Ya sure.

Just to clarify, it's preferred to use the keyword (ushort) over the system type name (UInt16) ya? So it's more readable